web/app/scripts/views/map/layers-controller.js

```diff
@@ -146,17 +146,23 @@
                 }
 
                 if (MapState.getFilterGeoJSON()) {
-                    var layer =  L.geoJson(MapState.getFilterGeoJSON());
+                    var layer = L.geoJson(MapState.getFilterGeoJSON());
                     layer.setStyle(filterStyle);
                     ctl.editLayers.addLayer(layer);
                 }
             });
         };
 
+        // Clears editLayers and informs the map state service
+        function clearLayers() {
```

Comment:
Might be good to name this to `clearEditLayers` or something, since it doesn't clear all the map layers.